### PR TITLE
Testnet is dead, long live testnet

### PR DESCRIPTION
**Pull request by bonomat**

### CODE REVIEW

File: .github/workflows/deliverables-push-main.yml [hunk 1]
```diff
@@ -63,9 +63,9 @@ jobs:
     secrets: inherit
     with:
       tag: main
-      esplora_endpoint: http://35.189.57.114:3000
-      coordinator_p2p_endpoint: 03507b924dae6595cfb78492489978127c5f1e3877848564de2015cd6d41375802@35.189.57.114:9045
+      esplora_endpoint: http://34.32.0.52:3000
+      coordinator_p2p_endpoint: 03507b924dae6595cfb78492489978127c5f1e3877848564de2015cd6d41375802@34.32.0.52:9045
       coordinator_port_http: 80
```
Comment (luckysori) [inline .github/workflows/deliverables-push-main.yml:68]: :mega: I think this needs to change too.

File: .github/workflows/deliverables-push-main.yml [hunk 2]
```diff
@@ -42,12 +42,12 @@ jobs:
     uses: ./.github/workflows/ios-fastlane.yml
     secrets: inherit
     with:
-      esplora_endpoint: http://35.189.57.114:3000
-      coordinator_p2p_endpoint: 03507b924dae6595cfb78492489978127c5f1e3877848564de2015cd6d41375802@35.189.57.114:9045
+      esplora_endpoint: http://34.32.0.52:3000
+      coordinator_p2p_endpoint: 03507b924dae6595cfb78492489978127c5f1e3877848564de2015cd6d41375802@34.32.0.52:9045
       coordinator_port_http: 80
```
Comment (luckysori) [inline .github/workflows/deliverables-push-main.yml:47]: :mega: I think this needs to change too.

**Review by luckysori — COMMENTED** [review]
LGTM, except for the coordinator's HTTP API port in a couple of places.

File: .github/workflows/deliverables-push-main.yml (hunk 1 above)
Comment (bonomat) [reply .github/workflows/deliverables-push-main.yml:68]: Thanks, it should be all on port `80` again.

**Review by luckysori — APPROVED** [review]

**Pull request merged**